charts/gatus/values.yaml

```diff
@@ -0,0 +1,42 @@
+gatus:
+  deployment:
+    annotateConfigChecksum: false
+
+  env:
+    GATUS_CONFIG_PATH: /configfiles
+
+  extraVolumeMounts:
+    - name: config
+      mountPath: /configfiles
+
+  extraInitContainers:
```

Comment:
We may need to run the config loader as an init container as wel as a sidecar container, because I notice that gatus often starts before the config load is finished.

Reply:
I already wanted you to ask about it, since I noticed the same